doc/SLSKPROTOCOL.md

```diff
@@ -67,31 +67,27 @@ If you find any inconsistencies, errors or omissions in the documentation, pleas
 | 1    | Away    |
 | 2    | Online  |
 
-### Transfer Direction
+### Transfer Directions
 
-| Code | Type     |
-| ---- | -------- |
-| 0    | Download |
-| 1    | Upload   |
+| Code | Direction          |
+| ---- | ------------------ |
+| 0    | Download from Peer |
+| 1    | Upload to Peer     |
 
 ### File Attribute Types
 
-| Code | Type             |
+| Code | Attribute (unit) |
 | ---- | ---------------- |
 | 0    | Bitrate (kbps)   |
 | 1    | Length (seconds) |
 | 2    | VBR (0 or 1)     |
 | 3    | Encoder (unused) |
 | 4    | Sample Rate (Hz) |
-| 5    | Bit Depth        |
+| 5    | Bit Depth (bits) |
 
-#### Used Attribute Combinations
+*File Attribute Combinations:*
 
-  - Soulseek NS, SoulseekQt (2015-2-21 and earlier), Nicotine+, Museek+, SoulSeeX, slskd (lossy formats):
-      - {0: *bitrate*, 1: *length*, 2: *VBR*}
-
-  - SoulseekQt (2015-2-21 and earlier):
-      - {0: *bitrate*, 2: *VBR*}
+These combinations are actively used by clients. More combinations are possible, of course.
```

Comment:
We shouldn't encourage new combinations unless the official client makes changes.